apps/dsb-client-gateway-api/src/app/modules/dsb-client/service/dsb-api.service.ts

```
@@ -1033,7 +1055,7 @@ export class DsbApiService implements OnApplicationBootstrap {
     } catch (e) {
       this.logger.error(`Get topic with topicId: ${topicId} failed`, e);
 
-      throw new Error(e);
+      return null;
```

shouldn't it throw an error instead of null like `throw e`